website/server/controllers/api-v3/user.js

```diff
@@ -1495,7 +1496,7 @@ api.userReleaseMounts = {
 api.userSell = {
   method: 'POST',
   middlewares: [authWithHeaders()],
-  url: '/user/sell/:type/:key',
+  url: '/user/sell/:type/:key/:amount',
```

Comment:
will this break old calls to /user/sell/:type/:key since it now expect a longer path?